libs/auth-api-lib/migrations/20241018150418-add-default-scopes-to-client-allowed-scope-table.js

```
@@ -0,0 +1,56 @@
+'use strict'
```

Biome

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)